docs/install_guides/installing-openvino-docker-windows.md

```diff
@@ -5,170 +5,249 @@ This guide provides steps for creating a Docker image with Intel® Distribution
 ## <a name="system-requirments"></a>System Requirements
 
 @sphinxdirective
-.. tab:: Target Operating System with Python Version
 
-  +------------------------------------+--------------------------+
-  | Operating System                   | Supported Python Version |
-  +====================================+==========================+
-  | Windows Server Core base LTSC 2019 | 3.8                      |
-  +------------------------------------+--------------------------+
-  | Windows 10, version 20H2           | 3.8                      |
-  +------------------------------------+--------------------------+
+.. tab:: Supported Operating Systems with Python Version
 
-.. tab:: Host Operating Systems
+  +------------------------------------+--------------------------+---------------------------------------------------------------------------------------------------+
+  | Target Operating System            | Python Version           | Host Operating System                                                                             |
+  +====================================+==========================+===================================================================================================+
+  | Windows Server Core base LTSC 2019 | 3.8                      | Windows 10 (64-bit) Pro, Enterprise, Education<br> (1607 Anniversary Update, Build 14393 or later)|
+  +------------------------------------+--------------------------+---------------------------------------------------------------------------------------------------+
+  | Windows 10, version 20H2           | 3.8                      | Windows Server 2016 or higher                                                                     |
+  +------------------------------------+--------------------------+---------------------------------------------------------------------------------------------------+
```

Comment:
Will this make it look like one target OS corresponds to one host system?

Reply:
Now, that you mention it, one might actually think they're corresponding. Guess, the best way is to leave these separate as they were. I have reverted the layout. Thank you.